app/jobs/regular/assign_notification.rb

```diff
@@ -4,6 +4,7 @@ module Jobs
   class AssignNotification < ::Jobs::Base
     def execute(args)
       raise Discourse::InvalidParameters.new(:assignment_id) if args[:assignment_id].nil?
+      return if SilencedAssignment.exists?(assignment_id: args[:assignment_id])
```

Comment:
The most minor style thing -- empty line under early return (guard clauses) is kinda nice.